backend/src/v2/metadata/client.go

```diff
@@ -978,6 +992,40 @@ func (c *Client) matchedArtifactOrNot(ctx context.Context, target *pb.Artifact,
 	return false, nil
 }
 
+// GetContexts ...
```

Comment:
Can we remove this comment?

Reply:
see [this comment](https://github.com/opendatahub-io/data-science-pipelines/pull/62#discussion_r1674668650)